libs/platform/core/src/services/page-meta/page-meta-resolver/page-meta-resolver.service.ts

```diff
@@ -0,0 +1,31 @@
+import { OnDestroy } from '@spryker-oryx/di';
+import { Observable } from 'rxjs';
+import { AppInitializer } from '../../app-initializer';
+import { ElementDefinition } from '../page-meta.model';
+
+export const PageMetaResolverService = 'oryx.PageMetaResolverService';
+export const PageMetaResolver = 'oryx.PageMetaResolver*';
+
+export interface ElementResolver extends Partial<ElementDefinition> {
+  link?: string;
+  style?: string;
+  script?: string;
+  title?: string;
+  id?: string;
+}
+
+export interface PageMetaResolverService extends AppInitializer, OnDestroy {
+  getTitle(): Observable<string | undefined>;
+}
+
+export interface PageMetaResolver {
+  getScore(): Observable<number>;
```

Comment:
can we make it optional? (assume it's `0` by default? or better 1, so normal?)

Also, what about some enum with -1 : not used, 0 : fallback, 1: default?